security-privacy.html

```diff
@@ -207,90 +207,117 @@
 <section class="informative"> <h2>Implementation details</h2>
   <p>
     This section contains discussion of implementation possibilities of security
-    related mechanisms.
+    related mechanisms. The following implementation alternatives are possible:
+    <ul>
+      <li>
+        Use additional NDEF records in an NDEF message containing
+        Web NFC specific metadata, such as identification, whitelists and
+        blacklists.
+      </li>
+        Use a specific Web NFC record format and encode the Web NFC specific
+        metadata into that.
+      </li>
+    </ul>
   </p>
-  <section> <h3>Web NFC tags</h3>
+  <section> <h3>Web NFC using additional NDEF records for metadata</h3>
+    <p>
+      A typical Web NFC NDEF message will contain the following
+      NDEF records:
+      <ul>
+        <li>Normal NDEF records carrying data.</li>
+        <li>
+          An additional NDEF record for identification.
```

Comment:
I would say "An additional NDEF Web NFC identification record" to make it clear what we are identifying